### PR TITLE
Un minify JS and check if window.onerror is actually needed to captur…

### PR DESCRIPTION
**Pull request by StupidCodeFactory**
…e error

### CODE REVIEW

File: app/assets/javascripts/modules/moj.sentry.js [hunk 1]
```diff
@@ -7,8 +7,12 @@
       this.raven = Raven;
       this.sentry_js_dsn = $(this.el).data('sentry-js-dsn');
       this.raven.config(this.sentry_js_dsn).install();
-      var sentry_module = this;
 
+      // Capture any uncaught errors
+      // window.onerror = $.proxy(this.handleException, this);
```
Comment (chrishanes) [inline app/assets/javascripts/modules/moj.sentry.js]: Is this meant to be commented out?

Comment (StupidCodeFactory) [inline app/assets/javascripts/modules/moj.sentry.js]: yes, i'm not convinced this is needed and it seems to obliterate the where the actual error is

Comment (StupidCodeFactory) [inline app/assets/javascripts/modules/moj.sentry.js]: twas in the PR title

Comment (chrishanes) [inline app/assets/javascripts/modules/moj.sentry.js]: So the handleException won't be called

Comment (StupidCodeFactory) [inline app/assets/javascripts/modules/moj.sentry.js]: @chrishanes  nope but i believe Raven will have some kind of mechanism to do it automatically

**Pull request merged**